apps/backend/src/websocket/server.py

```
@@ -0,0 +1,215 @@
+    2. Client sends auth message with Bearer token
+    3. Server authenticates and responds
+    4. Client can subscribe to topics
+    5. Server streams real-time data
+    """
+    client_id = await connection_manager.connect(websocket)
+    
+    try:
+        # Wait for authentication
+        auth_timeout = 30  # seconds
+        authenticated = False
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)